docs/src/callbacks.md

```diff
@@ -32,12 +32,71 @@ errors or user-specified integrals, and print the results to the screen. The res
 saved in a file. An example can be found at [examples/2d/elixir\_euler\_vortex.jl](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/2d/elixir_euler_vortex.jl).
 
 ### I/O
+
+#### Solution and restart files
 To save the solution in regular intervals you can use a [`SaveSolutionCallback`](@ref). It is also
 possible to create restart files using the [`SaveRestartCallback`](@ref). An example making use
 of these can be found at [examples/2d/elixir\_advection\_extended.jl](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/2d/elixir_advection_extended.jl).
 An example showing how to restart a simulation from a restart file can be found at
 [examples/2d/elixir\_advection\_restart.jl](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/2d/elixir_advection_restart.jl).
 
+#### Time series
+Sometimes it is useful to record the evoluation of state variables over time at
+a given set of points. This can be achieved by the [`TimeSeriesCallback`](@ref), which is used,
+e.g., in
+[examples/2d/elixir\_ape\_gaussian\_source.jl](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/2d/elixir_ape_gaussian_source.jl).
+The `TimeSeriesCallback` constructor expects a semidiscretization and a list of points at
+which the solution should be recorded in regular time step intervals. After the
+last time step, the entire record is stored in an HDF5 file.
+
+For the points, two different input formats are supported: You can either provide
+them as a list of tuples, which is handy if you specify them by hand on the
+REPL. Alternatively, you can provide them as a two-dimensional array, where the
+first dimension is the point number and the second dimension is the
+coordinate dimension. This is especially useful when reading them from a file.
+
+For example, to record the primitive variables at the points `(0.0, 0.0)` and
+`(-1.0, 0.5)` every five timesteps and storing the collected data in the file
+`tseries.h5`, you can create the `TimeSeriesCallback` as
+```julia
+time_series = TimeSeriesCallback(semi, [(0.0, 0.0), (-1.0, 0.5)];
+                                 interval=5,
+                                 solution_variables=cons2prim,
+                                 filename="tseries.h5")
+```
```

Comment:
Make this a doctest to check whether someone changes the constructor accidentally without modifying the docs?

Reply:
Are you sure? I thought about it briefly but then decided against it since I'd have to create semi first...